README.md

```diff
@@ -2,12 +2,20 @@
 
 Clouseau uses Scalang to expose Lucene functionality via erlang-like nodes.
 
-## Running a local dev cluster
+## Setup
+
+define the following environment variables:
+* `CLOUSEAU_NAME` (default = "clouseau@127.0.0.1")
```

Comment:
es sollte eigentlich primär mit commandozeilen argumenten gearbeitet werden, nicht env vars. 
wäre das auch möglich? nur mit env vars lässt sich keine testcluster in einem container oder auf der dev maschine starten

Reply:
Best practise ist mit Env vars zu arbeiten (12 Factor app)
Innerhalb eines Containers sollten sowieso keine multiplen Instanzen erstellt werden.
EInmal davon abgesehen, dass es schon grenzwertig ist clouseau innerhalb des couchdb containers zu verwenden.

Reply:
Es können argumente übergeben werden, diese wären allerdings nur durch ihre Reihenfolge identifizierbar. Ohne Launcher sind keine benamten Argumente möglich.